### PR TITLE
Add Echelon (ECH)

### PR DESCRIPTION
**Pull request by echelonfoundation**
Add Echelon (ECH) Chain ID: 3000

### CODE REVIEW

**Review by ligi — COMMENTED** [review]
Is this a rename of the other chain?

**Comment by echelonfoundation** [comment]
> Is this a rename of the other chain?

The other chain confirmed they only use this chain ID for their testnet. You can safely update the information to Echelon.

**Comment by ligi** [comment]
> The other chain confirmed they only use this chain ID for their testnet. You can safely update the information to Echelon.

do you have a link to the place where the other chain confirmed this?

**Comment by echelonfoundation** [comment]
> > The other chain confirmed they only use this chain ID for their testnet. You can safely update the information to Echelon.
> 
> do you have a link to the place where the other chain confirmed this?

It was in their Discord about a week ago, we can try to find the screenshot, just circling back around to this as Defilama would need the merge to add us.

**Comment by echelonfoundation** [comment]
> I need a confirmation of the old chain to merge. Even then I think it is a very bad idea as you expose your users to the risk of replay attacks. But at least I need a confirmation of the old chain - otherwise there will be no merge here.

Can you maybe ping the original committer? It is a testnet for them, so no issue.

**Comment by echelonfoundation** [comment]
@aidan-starke

**Comment by aidan-starke** [comment]
hey, yup this is one of our testnets so no problem for us

**Comment by echelonfoundation** [comment]
@ligi Confirmation above ^^

Thank you both!

**Comment by echelonfoundation** [comment]
Hello! We would like to get this merged in ASAP.

@ligi Please ack

**Comment by ligi** [comment]
Icon format must be a png or svg but was gif

**Comment by ligi** [comment]
also it is 5MB - very big and has no size annotation

**Comment by echelonfoundation** [comment]
> Icon format must be a png or svg but was gif

Updated to 5kb svg.

**Comment by ligi** [comment]
Please add icon width and heitght - also your faucet looks weird - is that really a faucet?

**Comment by echelonfoundation** [comment]
> Please add icon width and heitght - also your faucet looks weird - is that really a faucet?

Yes? Not sure if you are looking at our dapp and why the disrespectful comment...But to educate you: 

You can only use the faucet with 1 IP, ECH address, and device. No VPN or Proxy allowed. You also need to disable ad-blockers or Braves default blocker if you are visiting the faucet.

https://app.ech.network/faucet

We mostly help users through our Discord if they have issues with receiving funds from the faucet. You can purchase native ECH off exchanges now also.

You can find our full dapp at https://app.ech.network for delegating to validators, bridging from any of the seven other EVMs we support free of charge (just gas), etc.

The width and height were added. @ligi

File: _data/icons/echelon.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 [
   {
     "url": "ipfs://QmWgogVZKsEk3yk2h66P4qTzy9ujYG764RwsH6K6kGN7R4",
+    "width":128,
+	  "height":128,
```
Comment (ligi) [inline _data/icons/echelon.json:5]:
```suggestion
    "width":160,
    "height":235,
```

**Comment by ligi** [comment]
> https://app.ech.network/faucet

I see a different one in the diff:
image

Also my comment was not disrespectful - was just reviewing and saying what seems wrong. But in contrast I found your "But to educate you" really disrespectful. I really do not need your "education".
Also fixed your size change - that was wrong.

**Comment by echelonfoundation** [comment]
> > https://app.ech.network/faucet
> 
> I see a different one in the diff: image
> 
> Also my comment was not disrespectful - was just reviewing and saying what seems wrong. But in contrast I found your "But to educate you" really disrespectful. I really do not need your "education". Also fixed your size change - that was wrong.

You do need "education" then on how to speak to people properly. It was a rude and disrespectful way of wording it.  Which, you are still being disrespectful. Anyone will be able to read this PR and can see your responses, as well as ours. We really recommend that you take a look at this article on respect: https://www.voiceproinc.com/respect-a-communication-skill-that-brings-the-world-together/

There is a permanent redirect setup to go to https://app.ech.network/faucet and we updated the PR to include the correct subdomain.

Thank you!

**Comment by echelonfoundation** [comment]
@ligi PR should be ready for merging....

**Comment by echelonfoundation** [comment]
@ligi Still awaiting merge! Everything should be good to merge.

**Comment by echelonfoundation** [comment]
> @aidan-starke should we maybe revert to your testnet? See: #1216

They already confirmed it is their testnet. Our mainnet is live and working, with zero issues, bridge included, you can join the community and ask questions, we are very community oriented. https://discord.gg/echelon 

Why would you even post this, to revert us based off a post with no substance, that is literally shilling you a scam in the comments?

**Comment by ligi** [comment]
I really have no interest joining your discord. Continuing the discussion in the other issue. If the stuff raised there is confirmed I will just revert. A testnet is better than such a network ..

**Comment by echelonfoundation** [comment]
> > https://app.ech.network/faucet
> 
> I see a different one in the diff: image
> 
> Also my comment was not disrespectful - was just reviewing and saying what seems wrong. But in contrast I found your "But to educate you" really disrespectful. I really do not need your "education". Also fixed your size change - that was wrong.

Hey! Alert alert! 

@ligi https://github.com/ligi you have an impersonator, they almost got us.

**Pull request merged**